warehouse/query-core/src/main/java/datawave/query/index/lookup/ShardRangeStream.java

```diff
@@ -134,4 +137,33 @@ public QueryPlan apply(Entry<Key,Value> entry) {
             //  @formatter:on
         }
     }
+
+    /**
+     * Lift and shift from DefaultQueryPlanner to avoid reliance on static methods
+     */
+    private void configureTypeMappings(ShardQueryConfiguration config, IteratorSetting cfg, MetadataHelper metadataHelper) {
+        DefaultQueryPlanner.addOption(cfg, QueryOptions.QUERY_MAPPING_COMPRESS, Boolean.toString(true), false);
+
+        Multimap<String,Type<?>> nonIndexedQueryFieldsDatatypes = HashMultimap.create(config.getQueryFieldsDatatypes());
+        nonIndexedQueryFieldsDatatypes.keySet().removeAll(config.getIndexedFields());
+        String nonIndexedTypes = QueryOptions.buildFieldNormalizerString(nonIndexedQueryFieldsDatatypes);
+        DefaultQueryPlanner.addOption(cfg, QueryOptions.NON_INDEXED_DATATYPES, nonIndexedTypes, false);
+
+        try {
+            String serializedTypeMetadata = metadataHelper.getTypeMetadata(config.getDatatypeFilter()).toString();
+            DefaultQueryPlanner.addOption(cfg, QueryOptions.TYPE_METADATA, serializedTypeMetadata, false);
+        } catch (TableNotFoundException e) {
+            throw new RuntimeException(e);
+        }
+
```

Comment:
Could the above and below try statements be simplified into a single try since we end up throwing the same RTE regardless of no table or IO problems?